data/core/tips.json

```diff
@@ -362,16 +362,146 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Certain corpses can give you bionics if you dissect them." ]
+    "text": [ "Don't be too greedy.  Loot doesn't matter if you're dead." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Don't be too greedy.  Loot doesn't matter if you're dead." ]
+    "text": [ "The floor is too hard to sleep on?  Try gathering a pile of leaves." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "The floor is too hard to sleep on?  Try gathering a pile of leaves." ]
+    "text": [ "Too cold to sleep well at night?  A pile of clothes will make it better." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Lost much blood?  Drink to replenish fluids, but red blood cells need time to renew." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Arterial bleeding from a gunshot?  A tourniquet can save you from death." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "No bandages nearby?  Empty your hands and let the time pass to apply pressure to the wound." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Tis but a scratch?  Minor bleeding does not require any dressing.  Bandaid is sufficient enough." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Zone activity manager can make your life easier.  Do more in one go." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Want less micro-management?  Zone manager can change your game." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Every sensei was once a student.  NPCs can teach you and vice versa." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Length matters… at least when it comes to shoving a shovel in your backpack." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Where did my lighter go again?  Manage your containers to store specific items in them." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Keep essentials in your clothing pockets in case you have to ditch your backpack." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Be fit.  Exercising and being active increases your cardio and therefore your stamina pool." ]
```

Comment:
Recommend the first sentence be replaced with `Keep yourself fit.`

Reply:
I'll cut out the first sentence - it can't be any longer due to the menu width constraints, and the second sentence being solo will be enough.